mslice_main_window.py

```diff
@@ -0,0 +1,56 @@
+from PyQt4 import QtGui
+
+from MSliceMainView import MainView
+from workspacemanager.WorkspaceManagerPresenter import MainViewPresenter
+from command import Command
+from ui_Mslice import Ui_MainWindow
+
+class MSliceGui(MainView,QtGui.QMainWindow,Ui_MainWindow):
+    #TODO Should i set a metaclass?
```

Comment:
If you are allowed to create instances of this GUI, such that you can inherit from both `MainView` and `Ui_MainWindow` then no don't add it. I had to use the `metaclass` fix as a work around.